addons/event/models/event_event.py

```diff
@@ -469,6 +434,14 @@ def _compute_event_ticket_ids(self):
                 ]
             event.event_ticket_ids = command
 
+    @api.depends('event_type_id')
+    def _compute_note(self):
+        for event in self:
+            if event.event_type_id and event.event_type_id.note:
```

Comment:
Would use "not is_html_empty" as it is an html field that could hold dummy content

Reply:
Done